packages/dx-vue-demos/src/demo-sources/grid-data-types/none/editors.js

```diff
@@ -0,0 +1,146 @@
+import {
+  EditingState as DxEditingState,
+  DataTypeProvider as DxDataTypeProvider,
+} from '@devexpress/dx-vue-grid';
+import {
+  Grid as DxGrid,
+  Table as DxTable,
+  TableHeaderRow as DxTableHeaderRow,
+  TableEditRow as DxTableEditRow,
+  TableEditColumn as DxTableEditColumn,
+} from '@devexpress/dx-vue-grid-bootstrap4';
+import {
+  generateRows,
+  globalSalesValues,
+} from '../../../demo-data/generator';
+
+const BooleanFormatter = {
+  props: ['value'],
+  template: `
+    <span class="badge badge-secondary">{{ String(this.value) === 'true' ? 'Yes' : 'No' }}</span>
+  `,
+};
+
+const BooleanEditor = {
+  props: ['value'],
+  methods: {
+    onChange(event) {
+      this.$attrs.onValueChange(event.target.value);
+    },
+  },
+  template: `
+    <select
+      class="form-control"
+      :value="value"
+      v-on:change="onChange"
```

Comment:
Can we use `@change` as we do in other demos? For example [here](https://github.com/DevExpress/devextreme-reactive/pull/1002/files#diff-a7c36ccd3b35c6ae24634341f5626279R31).